### PR TITLE
new icon: Vuetify (original, line, plain)

### PR DESCRIPTION
**Pull request by vikash2806**
# **New icon section**

## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] The PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date
https://vuetifyjs.com/en/

### CODE REVIEW

File: devicon.min.css [hunk 1]
```diff
@@ -1 +1 @@
...
...ify-plain.colored,.devicon-vuetify-line.colored{color:#1697f6}.devicon-vscode-plain.colored,.devicon-vscode-plain-wordmark.colored{color:#3c99d4}.devicon-weblate-plain.colored,.devicon-weblate-plain-wordmark.colored{color:#2eccaa}.devicon-webpack-plain.colored,.devicon-webpack-plain-wordmark.colored{color:#1c78c0}.devicon-webstorm-plain.colored,.devicon-webstorm-plain-wordmark.colored{color:#2788b5}.devicon-windows8-original.colored,.devicon-windows8-original-wordmark.colored,.devicon-windows8-plain.colored,.devicon-windows8-plain-wordmark.colored{color:#00adef}.devicon-woocommerce-plain.colored,.devicon-woocommerce-plain-wordmark.colored{color:#7f54b3}.devicon-wordpress-plain.colored,.devicon-wordpress-plain-wordmark.colored{color:#494949}.devicon-xd-plain.colored,.devicon-xd-line.colored{color:#dd80bc}.devicon-yarn-plain.colored,.devicon-yarn-plain-wordmark.colored{color:#2c8ebb}.devicon-yii-plain.colored,.devicon-yii-plain-wordmark.colored{color:#0073bb}.devicon-yunohost-plain.colored{color:#fff}.devicon-zend-plain.colored,.devicon-zend-plain-wordmark.colored{color:#68b604}.devicon-zig-original.colored,.devicon-zig-plain-wordmark.colored,.devicon-zig-plain.colored{color:#f7a41d}
```
Comment (Panquesito7) [inline devicon.min.css:1]: Please add the endline back. Without this endline, there can be issues with some IDEs and configurations.

File: icons/vuetify/vuetify-line.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+        d="M31.5312 0L64 59.1034V59.5862L96 0H64H31.5312ZM48.1042 8.82759H64V37.5086L63.9271 37.6466L48.1042 8.82759ZM0 17.6552L64 128L128 17.6552H96L64 75.0345L32 17.6552H0ZM102.823 26.4828H111.052L64 107.603V96.1034L73.6875 78.7414L102.823 26.4828Z"
+        fill="black" />
+</svg>
```
Comment (Panquesito7) [inline icons/vuetify/vuetify-line.svg:5]: Missing an endline.
```suggestion
</svg>

```

File: icons/vuetify/vuetify-original.svg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M64.0039 0H34.1016L64.0039 61.8477" fill="#1697F6" />
+    <path d="M64.0039 80V128L0 18.6641H33.5273L64.0039 80ZM64.0039 0H93.9023L64.0039 61.8477" fill="#7BC6FF" />
+    <path d="M94.4727 18.6641H128L64.0039 128V80L94.4727 18.6641Z" fill="#AEDDFF" />
+</svg>
```
Comment (Panquesito7) [inline icons/vuetify/vuetify-original.svg:5]: Missing an endline.
```suggestion
</svg>

```

File: icons/vuetify/vuetify-plain.svg [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+        d="M34.4418 0L64.257 61.824L93.8795 0H34.4418ZM0 18.6667L64.257 128L128 18.6667H94.8434L64.257 80L33.3494 18.6667H0Z"
+        fill="black" />
+</svg>
```
Comment (Panquesito7) [inline icons/vuetify/vuetify-plain.svg:5]: Missing an endline.
```suggestion
</svg>

```

File: devicon.json [hunk 5]
```diff
@@ -5543,5 +5543,28 @@
         },
         "color": "#DC7B2E",
         "aliases": []
+    },
+    {
+        "name": "vuetify", 
+
```
Comment (Panquesito7) [inline devicon.json:5549]:
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Thanks for submitting a PR! 😄 Please revert the changes in the `package-lock.json`. Let us know if you need any help. Thanks. :)

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Could you please add the changes back in the `devicon.json` file? Thanks.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by Thomas-Boi — APPROVED** [review]
Great job @vikash2806! Just fyi, we will have a release within this month so you'll see your icon soon.

**Comment by vikash2806** [comment]
> Great job @vikash2806! Just fyi, we will have a release within this month so you'll see your icon soon.

Thank you , That's awesome. I'am looking forward to contribute more to Devicon.

**Pull request merged**